components/navigation/NavBar.tsx

```diff
@@ -144,106 +159,119 @@ export default function NavBar({ className = '', hideLogo = false }: NavBarProps
   }, [asPath]);
 
   return (
-    <div className={`bg-white ${className} z-50`}>
-      <div className='flex w-full items-center justify-between py-6 lg:justify-start lg:space-x-2'>
-        {!hideLogo && (
-          <div className='lg:w-auto lg:flex-1'>
-            <div className='flex'>
-              <Link href='/' className='cursor-pointer' aria-label='AsyncAPI' data-testid='Navbar-logo'>
-                <AsyncAPILogo className='w-auto' />
-              </Link>
+    <>
+      <div
+        // eslint-disable-next-line max-len
```

Comment:
You can run 'npm run lint' or use the VSCode eslint extension to fix this instead of disabling the check.